src/rdposix.h

```diff
@@ -30,7 +29,8 @@
 /**
  * POSIX system support
  */
-#pragma once
+#ifndef RDPOSIX_H
```

Comment:
It should solely be based on filename: make uppercase, prefix with `_`, change dot to underscore.
Whatever else I said is wrong :)